packages/frontend2/src/server/features/scaling/get-common-scaling-entry.ts

```diff
@@ -4,17 +4,45 @@ import {
   type StageConfig,
   layer2s,
 } from '@l2beat/config'
+import { ProjectId } from '@l2beat/shared-pure'
+import { type SetOptional } from 'type-fest'
 import { isAnySectionUnderReview } from './utils/is-any-section-under-review'
 
-export function getCommonScalingEntry({
-  project,
-  isVerified,
-  hasImplementationChanged,
-}: {
-  project: Layer2 | Layer3
-  isVerified: boolean
-  hasImplementationChanged: boolean
-}) {
+export function getCommonScalingEntry(
+  params:
+    | {
+        project: Layer2 | Layer3
+        isVerified: boolean
+        hasImplementationChanged: boolean
+      }
+    | { project: 'ethereum' },
+) {
+  if (params.project === 'ethereum') {
+    return {
+      id: ProjectId.ETHEREUM,
+      name: 'Ethereum',
+      shortName: undefined,
+      slug: 'ethereum',
+      type: undefined,
+      category: undefined,
+      provider: undefined,
+      purposes: [],
+      warning: undefined,
+      redWarning: undefined,
+      isVerified: true,
+      showProjectUnderReview: false,
+      isArchived: false,
+      hostChain: undefined,
+      href: undefined,
+      hasImplementationChanged: false,
+      isUpcoming: false,
+      isUnderReview: false,
+      stage: { stage: 'NotApplicable' as const },
+    }
```

Comment:
I understand the rationale, but don't we need that for activity only? If so, can we inject the Ethereum project into the activity entries generation step directly, or is it too intertwined with different utilities?
Question out of curiosity since I feel like it is somehow forcefully done here to make sure Ethereum is there for activity ;p

Reply:
We'll need this for one other page too + it's much easier to handle this there than in activity because of the ScalingFilters. They expect a CommonScalingEntry, which Ethereum will not conform to if it's not defined there (e.g. type is undefined).